bulk_migrate/README.md

```diff
@@ -1,9 +1,6 @@
 This script helps with bulk name migration
 
-Preparation:
-1. Update `Move.toml` with the correct `repository` account.
-
 Each iteration:
 1. Update the script with the appropriate names to migrate
 2. Compile the script: `aptos move compile`
-3. Run the script: `aptos move run-script --compiled-script-path bulk_migrate/build/bulk_migrate/bytecode_scripts/main.mv --profile admin`
+3. Run the script: `aptos move run-script --compiled-script-path bulk_migrate/build/bulk_migrate/bytecode_scripts/main.mv --profile admin bulk_migrate=address_of_user_initiating_migration`
```

Comment:
what should be address of `bulk_migrate`? should it be the address that deploys the bulk migrate contract to?